pagopa/pn-frontend-e2e
# PN-9874-e-2-e-fix-pipeline-pa

Issue Description
Some PA's tests are failing due to some id changed or just wrong feature steps

Changes Made:
- removed unused `feature steps`
- adjusted `webelements `and `ids`

How to Test
Simply run any test and verify if all the tests run fine.
Also a run into AWS CodeBuild is necessary in order to test the full everything.
